codes/docker-compose.yml

```diff
@@ -1,7 +1,12 @@
 version: '3.8'
 services:
   hello-algo-code:
-    build: .
+    build:
+      context: .
+      args:
+        # 设置需要安装的语言，使用空格隔开
+        # Set the languages to be installed, separated by spaces
+        LANGS: "python cpp csharp"
```

Comment:
I prefer setting the LANGS here with out adding a .ENV file. How do you think of it?

Reply:
remove env file